subsys/nrf_security/src/drivers/cracen/cracenpsa/src/key_management.c

```diff
@@ -1493,7 +1493,7 @@ psa_status_t cracen_copy_key(psa_key_attributes_t *attributes, const uint8_t *so
 
 	int sx_status;
 	int nested_err;
-	psa_status_t psa_status;
+	psa_status_t psa_status = PSA_ERROR_GENERIC_ERROR;
```

Comment:
```suggestion
	psa_status_t psa_status = PSA_ERROR_HARDWARE_FAILURE;
```

It does not matter as this can't be reached, but it makes it easier to understand what the issue is.

Reply:
Thanks, changed.